src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -139,6 +139,19 @@ private fun visualize(strategyObject: Any) = runCatching {
     visualizeInstance(testObject, objectToNumberMap, continuationToLincheckThreadIdMap, threadToLincheckThreadIdMap)
 }
 
+/**
+ * This method is called from the trace-debugger evaluation.
```

Comment:
Please indicate when the plugin calls this method.

Reply:
I have no idea when it happens)

Reply:
Please investigate and leave a comment